main.go

```diff
@@ -76,8 +76,7 @@ func main() {
 		EnvVar: "PRODUCER_TOPIC",
 	})
 
-	log.SetFormatter(&log.JSONFormatter{TimestampFormat: time.RFC3339Nano})
-	log.SetLevel(log.InfoLevel)
+	log.InitDefaultLogger(appSystemCode)
 	log.Info("[Startup] pac-annotations-mapper is starting ")
```

Comment:
Should this use the new `ServiceStarted` func?

Reply:
I used this [PR](https://github.com/Financial-Times/annotations-mapper/pull/2) as a reference for which kind of events we need to keep track. Even if the startup event is defined in the `go-logger` library, it is not a a kind of event that the monitoring system keeps track.